src/test/java/seedu/address/testutil/event/TypicalIndexes.java

```diff
@@ -0,0 +1,8 @@
+package seedu.address.testutil.event;
```

Comment:
Typical indexes are already available as a class in the testutil package. Since test cases for other command classes all use the index fields from `TypicalIndex` in the testutil package, I think you should use it too instead of creating a standalone TypicalIndex class specially for Event.